README.ui-testing.md

```diff
@@ -0,0 +1,91 @@
+# Ghost Inspector Testing
+
+Run [Ghost Inspector](http://ghostinspector.com/) tests on your `mh-opsworks` cluster using the `./bin/mh_ghost` script. This script acts as a wrapper around the ghost inspector commands provided by the
```

Comment:
In fact, there is! We could export them in Selenium format and another org could import from same. I haven't tried this to see how the use of test variables are preserved, particularly at the suite level. Selenium doesn't have the notion of a "suite", I don't think.